geode-core/src/main/java/org/apache/geode/cache/wan/GatewaySender.java

```diff
@@ -474,46 +474,4 @@
    *
    */
   boolean getEnforceThreadsConnectSameReceiver();
-
-
-  /**
-   * Set AlertThreshold for this GatewaySender.
-   *
-   * @since Geode 1.14
-   *
-   */
-  void setAlertThreshold(int alertThreshold);
```

Comment:
this looks like a public class, can we remove these APIs?

Reply:
the @since comment was incorrect, these were added in 1.15 so we still have a chance to remove them before we do ship 1.15